core/src/main/scala/cats/data/NonEmptyMap.scala

```diff
@@ -23,246 +23,261 @@ import cats.{Always, Apply, Eval, Foldable, Functor, Later, NonEmptyTraverse, No
 
 import scala.collection.immutable._
 
-final class NonEmptyMap[K, A] private(val value: SortedMap[K, A]) extends AnyVal {
-
-  private implicit def ordering: Ordering[K] = value.ordering
-  private implicit def order: Order[K] = Order.fromOrdering
-
-  /**
-    * Alias for [[concat]]
-    */
-  def ++(as: NonEmptyMap[K, A]): NonEmptyMap[K, A] = concat(as)
-  /**
-    * Appends this NEM to another NEM, producing a new `NonEmptyMap`.
-    */
-  def concat(as: NonEmptyMap[K, A]): NonEmptyMap[K, A] = new NonEmptyMap(value ++ as.value)
-
-  /**
-    * Removes a key from this map, returning a new SortedMap.
-    */
-  def -(key: K): SortedMap[K, A] = value - key
-
-  /**
-    * Adds a key-value pair to this map, returning a new `NonEmptyMap`.
-    * */
-  def +(ka: (K, A)): NonEmptyMap[K, A] = new NonEmptyMap(value + ka)
```

Comment:
I had to get rid of this operator because of the super annoying `any2StringAdd` :(